packages/ohm-js/src/ohm-grammar.ohm

```diff
@@ -61,6 +70,12 @@ Ohm {
     | terminal                                       -- terminal
     | "(" Alt ")"                                    -- paren
 
+  include
+  	= "include"
+    
+  relativeFilePath 
```

Comment:
What's the reason for choosing to allow these specific characters? Maybe I'm wrong, but it seems like it may be overly restrictive.

An alternative would be for Ohm to be completely agnostic about paths, and accept any character inside the quotes.

Reply:
This is following security standards to always use an allowlist when you know the expected content format. If you want to accept the risk and open it to any character between the quoutes we can do that.